fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/AssertNumRowsElement.java

```diff
@@ -75,7 +75,7 @@ public AssertNumRowsElement withChildren(List<Expression> children) {
     public String toString() {
         return Utils.toSqlString("desiredNumOfRows: ",
                 Long.toString(desiredNumOfRows),
-                "assertion: " + assertion);
+                "assertion: ", assertion);
```

Comment:
the first parameter is plan name, and followed by k-v pairs.

Reply:
done